package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "simple-schema-typescript",
+  "version": "1.0.0",
+  "description": "Schema Salad TypeScript-codegen draft for simple-schema",
+  "main": "index.js",
+  "scripts": {
+    "test": "nyc --reporter=lcov mocha --require ts-node/register src/test/**/*.ts"
+  },
+  "author": "Adrian Zimmer",
+  "license": "ISC",
```

Comment:
I think this was a copy-paste error :slightly_smiling_face: ? Other CWL projects are Apache 2.0, and the repo has a LICENSE file with the Apache license.

Reply:
Absolutely.